packages/turf-polygonize/lib/Graph.ts

```diff
@@ -205,7 +207,7 @@ export default class Graph {
 
       if (de.label === label) outDE = de;
 
-      if (sym.label === label) inDE = sym;
+      if (((sym as Edge).label as number) === label) inDE = sym;
```

Comment:
I really don't like using `as` across the code here but I wanted to change the logic as little as possible whilst still fixing the TypeScript issues